plutus-ledger-api/test-plugin/Spec/Data/ScriptContext/9.6/succeedsIfHasDatum.uplc.golden

```diff
@@ -26,11 +26,11 @@
                                (force headList
                                   (force tailList
                                      (force (force sndPair) tup)))))
-                         [(\ds -> constr 0 []), (constr 1 [])]) ])
+                         [(\ds -> True), False]) ])
                  (unConstrData
                     (force headList
                        (force tailList
                           (force tailList
                              (force (force sndPair) (unConstrData d)))))))
-              [(delay ()), (delay (traceError "PT5"))])
+              [(delay (traceError "PT5")), (delay ())])
```

Comment:
This is funny. The reason why those `delay`s are not optimized away is that `force` can't see past a function application that we have there. The force-delay optimization understands that, but the force-case-delay one doesn't. This is one reason why I prefer to merge optimization together instead of keeping them separate.